Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -168,6 +181,10 @@ theorem lt_of_mul_lt_mul_of_le₀ (h : a * b < c * d) (hc : 0 < c) (hh : c ≤ a
   simpa [inv_mul_cancel_left₀ ha, inv_mul_cancel_left₀ hc.ne']
     using mul_lt_mul_of_le_of_lt_of_nonneg_of_pos hh  h zero_le' (inv_pos.2 hc)
 
+theorem mul_lt_mul_right₀ {a b c : α}
+    (hc : 0 < c) : a * c < b * c ↔ a < b := by
+  simpa only [mul_comm a, mul_comm b] using mul_lt_mul_left hc
```

Comment:
Doesn't `mul_lt_mul_right` apply here?

Reply:
Indeed!

Reply:
Sorry, but why are you still adding this lemma then? Isn't it exactly a specialisation of `mul_lt_mul_right`? 😅

Reply:
Oh sure, I just implemented your suggestion without thinking what it meant... 🤭